### PR TITLE
HnH: use query to initialize sprint 

### PR DESCRIPTION
**Pull request by rvanderp3**

### CODE REVIEW

File: cmd/sprint/move-to-sprint.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	}
+
+	if sprint == nil {
+		return fmt.Errorf("unable to find sprint %s", sprintNumber)
+	}
+
+	log.Printf("found sprint %s\n", sprintNumber)
+
+	if applyFlag {
+		log.Printf("moving issues in to sprint\n")
```
Comment (fabianofranz) [inline cmd/sprint/move-to-sprint.go:83]: Maybe worth checking if a Sprint is set already, and have a flag like `--override`.

File: cmd/sprint/sprint.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+var cmdSprint = &cobra.Command{
+	Use:   "sprint",
+	Short: "Manage sprints",
+	Long:  `This command allows you to manage sprints in your project management tool.`,
+}
+
+func Initialize(rootCmd *cobra.Command) {
+
+	cmdMoveIssue.Flags().BoolVarP(&applyFlag, "apply", "a", false, "Apply the changes")
```
Comment (fabianofranz) [inline cmd/sprint/sprint.go]: `--dry-run` is kind of a standard for this kind of things, but I agree we probably want a dry run as the default behavior.

File: cmd/main.go [hunk 3]
```diff
@@ -2,109 +2,30 @@ package main
...
-		if err != nil {
-			fmt.Printf("unable to update epic %s: %v\n", issue.Key, err)
-			b := []byte{}
-			resp.Response.Body.Read(b)
-			fmt.Println(string(b))
-			continue
-		}
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
```
Comment (fabianofranz) [inline cmd/main.go:28]: nit: `log.Fatalf("Error: %s", err)`

File: cmd/sprint/move-to-sprint.go [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+	}
+
+	return err
+}
+
+var cmdMoveIssue = &cobra.Command{
+	Use:   "move-issue [sprint-number] [issue-number]...",
+	Short: "Move an issue to a sprint",
+	Long:  `This command allows you to move one or more issues to a sprint in your project management tool.`,
+	Args:  cobra.MinimumNArgs(2), // Requires exactly two arguments: sprint-number and issue-number
```
Comment (fabianofranz) [inline cmd/sprint/move-to-sprint.go]: If you expect exactly 2 args, you probably want `cobra.ExactArgs(2)`.

File: cmd/sprint/move-query-to-sprint.go [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+	"github.com/openshift-splat-team/splat-jira-bot/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+var cmdMoveInQuery = &cobra.Command{
+	Use:   "move-in-query [sprint-number] [JQL query]...",
+	Short: "Moves all issues returned by a JQL query",
+	Long:  `This command allows you to remove an issue from a sprint in your project management tool.`,
+	Args:  cobra.MinimumNArgs(2), // Requires exactly two arguments: sprint-number and issue-number
```
Comment (fabianofranz) [inline cmd/sprint/move-query-to-sprint.go]: Same.

File: cmd/sprint/move-to-sprint.go (hunk 4 above)
Comment (rvanderp3) [reply cmd/sprint/move-to-sprint.go]: for sure, in this case the number of issues is variable

**Pull request merged**